tests/e2e/tools/FFI/ContainerFile.tools

```diff
@@ -0,0 +1,20 @@
+FROM quay.io/centos/centos:stream9
+RUN dnf install -y \
+	dnf-plugin-config-manager \
+	epel-release
+RUN dnf config-manager -y --set-enabled crb
+
+RUN dnf install -y \
+        gcc
```

Comment:
probably good to add some editor there too, like vim, in case we need to edit the sources.

Reply:
sure

Reply:
It will big fat vm :) 
OK will add it